static/js/rich_search/app.tsx

```diff
@@ -131,14 +151,15 @@ function App({
 async function getStatVars(
```

Comment:
please add documentation for new functions, especially those which are exported.